Улучшение перекраски монстров

PROPHESSOR opened the pull request:
Review:
### DEADSKULL1
- [ ] Druj
- [x] Belphegor
- [ ] Commando
- [x] Zombie Lt
### PROPHESSOR
- [x] Zombie Cpt
- [x] Rahovart
- [x] Fiend

PROPHESSOR committed:
[DEADSKULL1] Добавлена предсмертная перекраска монстров (14.09.2019)

- Druj
- Belphegor
- Commando
- Zombie Lt

PROPHESSOR commented:
До:
![image](https://user-images.githubusercontent.com/19251320/69493209-ac8c4f00-0eb4-11ea-897c-3ac59229be3e.png)
![image](https://user-images.githubusercontent.com/19251320/69493215-b746e400-0eb4-11ea-827c-23328bae8fba.png)
![image](https://user-images.githubusercontent.com/19251320/69493219-c332a600-0eb4-11ea-9ce8-5f51f5ade81a.png)
![image](https://user-images.githubusercontent.com/19251320/69493202-a26a5080-0eb4-11ea-91d8-9c1d3ceb8186.png)

PROPHESSOR commented:
После:
![image](https://user-images.githubusercontent.com/19251320/69493228-f5dc9e80-0eb4-11ea-962f-5d9487c06635.png)
![image](https://user-images.githubusercontent.com/19251320/69493229-f8d78f00-0eb4-11ea-8244-4b2191a79999.png)
![image](https://user-images.githubusercontent.com/19251320/69493231-fbd27f80-0eb4-11ea-8c8c-1693c3cc6834.png)
![image](https://user-images.githubusercontent.com/19251320/69493232-fecd7000-0eb4-11ea-818e-9048a47c5efb.png)

PROPHESSOR commented:
Мне, если честно, из этого нравится только Belphegor (Синий бык), возможно ещё Zombie Lt (Синий зомби)
У Druj'я и Commando пропала кровь, у последнего, так же, появились синие артефакты на руке

PROPHESSOR committed:
Перекрасил Zombie Cpt

PROPHESSOR commented:
В рамках реколора перекрасил Zombie Cpt (в оригинале он мало отличается от Pvt, разве что, кровавыми подтёками)
![image](https://user-images.githubusercontent.com/19251320/69884423-0344bf00-12e1-11ea-9b93-baa908540ddc.png)

PROPHESSOR committed:
Подправил расцветку Rahovart'а

PROPHESSOR commented:
Так же, подправил Rahovart'а
До:
![image](https://user-images.githubusercontent.com/19251320/69884718-5ec37c80-12e2-11ea-9cc8-7b64af5bb4d7.png)
После:
![image](https://user-images.githubusercontent.com/19251320/69884725-671bb780-12e2-11ea-8f67-0ebcbcd19067.png)
![image](https://user-images.githubusercontent.com/19251320/69884857-06d94580-12e3-11ea-9740-6f73605f0b90.png)
Оригинал:
![image](https://vignette.wikia.nocookie.net/doom/images/b/bf/Rahovert.jpg/revision/latest?cb=20170113093749)

PROPHESSOR committed:
Подправил перекраску Fiend'а

Сделал менее блестащим и подправил цвет штанов

PROPHESSOR commented:
Подправил перекраску Fiend'а, сделал менее блестящим и подправил цвет штанов
До:
![image](https://user-images.githubusercontent.com/19251320/69885026-e067da00-12e3-11ea-9ae9-4f39854b1a1b.png)
После:
![image](https://user-images.githubusercontent.com/19251320/69885073-0beac480-12e4-11ea-8e76-beba76204d50.png)
![image](https://user-images.githubusercontent.com/19251320/69885077-10af7880-12e4-11ea-9c52-e4aa5fa380ac.png)
Оригинал:
![image](https://vignette.wikia.nocookie.net/doom/images/7/7f/Fiend.jpg/revision/latest?cb=20170113065542)

MorthimerMcMare commented:
Druj: оригинал от перекраски практически не отличаются, за исключением, как уже говорили, отсутствия крови. По мне, так оригинал лучше -- на верхних конечностях лучше видны следы, так сказать, насильственной кибернетизации организма.
Belphegor: определённо стал лучше.
Commando: чего-то не помню я, чтобы в DRPG у него голубые глаза были... А рот -- вообще пальцем в небо.
Zombie Lt: я теперь знаю, как выглядит отморозок! Странно, что на базе UAC всех лейтенантов одновременно послали на северный полюс без скафандров или каких-нибудь элементарных средств защиты... Да, дробовик в руках тоже в лёд превратился; огнестрельная ледышка -- что-то новенькое.

Zombie Cpt: слишком кроваво -- по сравнению с рядовым, у которого никаких кровоподтёков за теоретически длинное нахождение на базе не осталось, выглядит довольно странно. Можно даже сказать, необъяснимо. Ещё: во время анимации совершенно разные части автомата в руках кровью заливаются.
Rahovart: на оригинал похож, но для нормальной, уже не усечённой палитры Doom, не подходит.
Fiend: аналогично Раховарту.

MorthimerMcMare commented:
Последних двух -- принимаю в качестве возврату к оригиналу, но они мне не нравятся.

PROPHESSOR commented:
@MorthimerMcMare 
> Zombie Lt: я теперь знаю, как выглядит отморозок! Странно, что на базе UAC всех лейтенантов одновременно послали на северный полюс без скафандров или каких-нибудь элементарных средств защиты... Да, дробовик в руках тоже в лёд превратился; огнестрельная ледышка -- что-то новенькое.

Да, соглашусь, на замороженного похож :joy: 
Но цветовая схема, в принципе, похожа на ту, что была в оригинале (причём, намного больше, чем та, что есть сейчас)
![image](https://vignette.wikia.nocookie.net/doom/images/1/1b/ZombieLT.jpg/revision/latest?cb=20170113003600)

PROPHESSOR commented:
@MorthimerMcMare 
> Rahovart: на оригинал похож, но для нормальной, уже не усечённой палитры Doom, не подходит.
Fiend: аналогично Раховарту.

Лучше пока ничего предложить не могу, но эта перекраска исправляет сверкающую лысину Финда и водно-кисельный цвет Раховарта... Так что, я за применение этого фикса и надежду на нормальную перекраску в будущем

PROPHESSOR commented:
Т.е. как я понял, сейчас ситуация следующая:

- Druj - Оставить оригинал
- Belphegor - Применить
- Commando - Оставить оригинал
- Zombie Lt - *Оставить оригинал?*
- Zombie Cpt - Уменьшить количество крови, подправить оружие и применить
- Rahovart - Применить
- Fiend - Применить

Верно?

MorthimerMcMare commented:
> Но цветовая схема, в принципе, похожа на ту, что была в оригинале (причём, намного больше, чем та, что есть сейчас)

Да, с этим не поспоришь. Здесь уже упирается всё в палитру -- что больше нужно, оригинальность или невырвиглазность? Можно сделать, по-идее, чтобы верная перекраска от CVar-переменной зависела.

* * *

> PR: Лучше пока ничего предложить не могу, но эта перекраска исправляет сверкающую лысину Финда и водно-кисельный цвет Раховарта... Так что, я за применение этого фикса и надежду на нормальную перекраску в будущем

> MM: Последних двух -- принимаю в качестве возврату к оригиналу, но они мне не нравятся.

* * *

> Zombie Lt - _Оставить оригинал?_

Похож, похож на DRPG'шного... Давай тогда, оставляй -- буду драться против тех, у кого дуло огнестрельного оружия даже при критических температурах не меняет своей промороженной структуры. Пусть пока так будет, я тоже за надежду на нормальную перекраску в будущем.

MorthimerMcMare reviewed (changes requested):
Применяем всех, кроме Друйя и зомби-командира.

Все ремарки (и даже небольшой спор) -- в комментариях.

PROPHESSOR commented:
> зомби-командира.

Ты же имел в виду Commando, верно? :grinning:

PROPHESSOR committed:
Отменены изменения в перекраске монстров #15

- Убрана перекраска Druj от DEADSKULL1
- Убрана перекраска Commando от DEADSKULL1

Подробнее в PR #15
